libmscore/text.cpp

```diff
@@ -2659,7 +2659,12 @@ Element* Text::drop(const DropData& data)
                         }
                   else {
                         startEdit(data.view, data.pos);
-                        curLine().insert(_cursor, QChar(code));
+                        if (QChar::requiresSurrogates(code)) {
```

Comment:
I can't find a way to exercise it either. FSymbol are only used in the text palette (F2) and it's not possible to open the palette if not in edit mode.